include/swift/AST/Attr.def

```diff
@@ -496,6 +496,10 @@ DECL_ATTR(_projectedValueProperty, ProjectedValueProperty,
   OnVar | UserInaccessible |
   ABIStableToAdd | ABIStableToRemove | APIStableToAdd | APIStableToRemove,
   89)
+SIMPLE_DECL_ATTR(_nonEphemeral, NonEphemeral,
+  OnParam | UserInaccessible |
+  ABIStableToAdd | ABIStableToRemove | APIStableToAdd | APIStableToRemove,
```

Comment:
Not exactly APIStableToAdd, since it's going to produce an error.

Reply:
@jrose-apple I was planning on changing it once we errored by default, I guess there's no harm in doing it now though, I'll go ahead and make that change.